ios/StatusIm/StatusIm.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
What is this file?

Reply:
Necessary to turn on push notifications capabilities/entitlements as far as I can tell. Not present in any RN gitignore I've found. I believe it'll be fleshed out with production certificates.

Reply:
:+1: